src/components/NewLocationPage/SummaryStatsBlock/SummaryStatsBlock.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import SummaryStat from './SummaryStat';
+import { BlockWrapper } from './SummaryStatsBlock.style';
+import { Metric } from 'common/metricEnum';
+import { MetricValues } from 'common/models/Projections';
+import { orderedStatMetrics } from './utils';
+
+const SummaryStatsBlock: React.FC<{ stats: MetricValues }> = ({ stats }) => {
+  return (
+    <BlockWrapper>
+      {orderedStatMetrics.map((metric: Metric) => {
+        const value = stats[metric] as number;
+        return <SummaryStat metric={metric} value={value} />;
```

Comment:
We need to add a `key` prop here since we are rendering something in the body of a `map`